xformers/components/attention/csrc/cuda/mem_eff_attention/iterators/predicated_tile_iterator_residual_last.h

```diff
@@ -337,6 +339,11 @@ class PredicatedTileIterator<
     address_iterator_.clear_mask(enable);
   }
 
+  CUTLASS_HOST_DEVICE
+  void set_residual_tile(bool enable) {
+    address_iterator_.set_residual_tile(enable);
```

Comment:
It's a pity that `address_iterator` is a private member, otherwise we could have just inherited from the base class and added this method to all the classes in this file.